guide/src/commands/keys/index.md

```diff
@@ -122,6 +122,15 @@ To restore a key from its mnemonic:
 hermes -c config.toml keys restore [CHAIN_ID] -m "[MNEMONIC]"
 ```
 
+or using an explicit [derivation path](https://github.com/satoshilabs/slips/blob/master/slip-0044.md), for example
+an Ethereum coin type (used for Evmos, Injective, Umee, Cronos, and
+possibly other networks):
+
+```shell
+hermes -c config.toml keys restore --mnemonic <MNEMONIC> --hd-path "m/44'/60'/0'/0/0" <CHAIN_ID>
```

Comment:
Would it make sense to have hermes use the right derivation path based on the chain config `address_type = { derivation = 'evmos', ...`?

Reply:
This is a good UX improvement, I like the idea!

Not sure if (at this point) we should make it a priority, so let's capture this into an issue. I can handle that.